docs/pages/reference/helm-reference/includes/zz_generated.teleport-kube-agent.mdx

```diff
@@ -0,0 +1,1627 @@
+
+## `roles`
+
+| Type | Default |
+|------|---------|
+| `string` | `"kube"` |
+
+`roles` is a comma-separated list of services which should be enabled
+when running the `teleport-kube-agent` chart.
+
+| Services                     | Value for `roles` | Mandatory additional settings for this role |
+|------------------------------|-------------------|---------------------------------------------|
+| Teleport Kubernetes service  | `kube`            | [`kubeClusterName`](#kubeclustername)       |
+| Teleport Application service | `app`             | [`apps`](#apps) or [`appResources`](#appresources) |
+| Teleport Database service    | `db`              | [`databases`](#databases) or [`databaseResources`](#databaseresources) |
+| Teleport Discovery service   | `discovery`       | [`kubeClusterName`](#kubeclustername) |
+
+For example: `roles: kube,app,discovery`.
+
+## `proxyAddr`
+
+| Type | Default |
+|------|---------|
+| `string` | `""` |
+
+`proxyAddr` provides the public-facing Teleport proxy endpoint which should be used to join the cluster. This is the same URL that is used
+to access the web UI of your Teleport cluster. It is the same as the value configured for `proxy_service.public_addr` in a traditional
```

Comment:
Teleport Cloud users won't necessarily think of `proxy_service.public_addr` as traditional, so I'd find another way to put this.